python/cusignal/_upfirdn.py

```diff
@@ -351,33 +520,29 @@ def upfirdn(h, x, up=1, down=1, axis=-1, use_numba=True):
         The axis of the input data array along which to apply the
         linear filter. The filter is applied to each subarray along
         this axis. Default is -1.
-
+    use_numba : bool, optional
+        Option to use Numba CUDA kernel or raw CuPy kernel. Raw CuPy
+        can yield performance gains over Numba. Default is True.
     Returns
```

Comment:
We need to add documentation for cp_stream (in a similar way we added documentation for `use_numba`)

Reply:
How's this?
```
cp_stream : CuPy stream, optional
		Option allows upfirdn to run in a non-default stream. The use
		of multiple non-default streams allow multiple kernels to 
		run concurrently. Default is cp.cuda.stream.Stream(null=True)
		or default stream.
```